api-report/merge-tree.api.md

```diff
@@ -956,28 +907,22 @@ export function refTypeIncludesFlag(refPosOrType: ReferencePosition | ReferenceT
 // @public (undocumented)
 export const reservedMarkerIdKey = "markerId";
 
-// @public (undocumented)
+// @internal (undocumented)
```

Comment:
i think the reserved keys are used by consumers, as they set those properties in segment annotations

Reply:
you can see that in examples

Reply:
I only found references to `reservedMarkerIdKey` in the fluid-bohemia repo when I looked, but it makes sense to be more conservative with these constants -- I was debating whether to remove them for some time. I've removed the `@internal` marker from all of them, though I think only 2 have usage in examples.